pkg/services/rendering/rendering.go

```diff
@@ -110,6 +113,23 @@ func (rs *RenderingService) RenderErrorImage(err error) (*RenderResult, error) {
 }
 
 func (rs *RenderingService) Render(ctx context.Context, opts Opts) (*RenderResult, error) {
+	startTime := time.Now()
+	result, err := rs.render(ctx, opts)
+	elapsedTime := time.Since(startTime).Milliseconds()
+	if status.Code(err) == codes.DeadlineExceeded {
+		metrics.MRenderingRequestTotal.WithLabelValues("canceled").Inc()
+		metrics.MRenderingSummary.WithLabelValues("canceled").Observe(float64(elapsedTime))
```

Comment:
Nit, after considerations think status should be `timeout` instead since canceled I get the impression that it was canceled by the user. What do you think?

```suggestion
		metrics.MRenderingRequestTotal.WithLabelValues("timeout").Inc()
		metrics.MRenderingSummary.WithLabelValues("timeout").Observe(float64(elapsedTime))
```